.github/workflows/actions/auto-request-reviews/action.yml

```diff
@@ -0,0 +1,16 @@
+name: Auto Request Review
+
+inputs:
+  token:
+    description: 'GitHub token'
```

Comment:
For each of these descriptions, can we add additional detail? Especially for GitHub nomenclature like 'organization' and 'team slug', a little more detail + an explanation can go a long way for someone looking at this for the first time/needing to maintain it